### ISSUE TITLE
Typescript problem with knex.raw inside returning() function

### ISSUE THREAD

**Issue by selarep** [knex/knex#4879]
# Environment

Knex version: 0.95.14
Database + version: PostgreSQL 13
OS: Linux Mint 20.1

@lorefnon

# Bug

When I use knex.raw() inside a returning() statement, Typescript is throwing an error.
Here there is a code snippet as example:
knex
  .table('geometry_points')
  .insert(
    this.knex.raw(
      '(id, geo_point) VALUES (?, ST_GeomFromGeoJson(?))',
      [id, geoPoint]
    )
  )
  .returning([
  'id',
  knex.raw('ST_AsGeoJson(geo_point)::jsonb AS geo_point'), // <------ HERE IS THE ERROR
  ])
  .then(() => {
    // blah blah blah
  })
.

The error message I get is: The type 'Raw<any>' cannot be assigned to type 'string'.
If i try to write all the returning statement with a knex raw i get the same error. Also, if I I transform the line:
`knex.raw('ST_AsGeoJson(geo_point)::jsonb AS geo_point'),`
into this
`knex.raw('ST_AsGeoJson(geo_point)::jsonb AS geo_point') as any,`
then, everything is working as normal, so the api works like a charm, and it is only a problem with typings.

**Comment by OlivierCavadenti** [comment]
As you say, the API work with raw but is typing error here:

```js
returning<TResult2 = SafePartial<TRecord>[]>(
  column: string | readonly string[],
  options?: DMLOptions
): QueryBuilder<TRecord, TResult2>;
```

Pretty quick to fix (btw some tests need to be added)

**Comment by kibertoad** [comment]
Released in 1.0.1.
